pytest_mpi.py

```diff
@@ -174,6 +178,18 @@ def _set_parallel_callback(item):
     if nprocs == 1:
         return
 
+    # Set the executable by sniffing sys.argv[0]
```

Comment:
Please explain and leave a link explaining why these aren't equivalent things to do.